web/beacon-app/src/features/topics/components/TopicQueryResult/QueryResultContent.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from '@lingui/macro';
+import React from 'react';
+
+import DisplayResultData from './DisplayResultData';
+interface QueryResultContentProps {
+  result: any;
+  mimeType: string;
+}
+
+const QueryResultContent: React.FC<QueryResultContentProps> = ({ result, mimeType }) => {
+  console.log('[] result', result); // added this avoid eslint error
+  console.log('[] mimetype', mimeType);
+
+  return (
+    <div className="shadow-md h-20 max-h-80 overflow-y-auto bg-black p-4 text-white">
+      <pre className="mx-auto text-sm">
+        {result && <DisplayResultData result={result} mimeType={mimeType} />}
+        {!result && t`No results found`}
```

Comment:
I noticed that this message and the one below currently display in the screenshot that was provided. Shouldn't only one of them appear?

Reply:
Good catch, I improve this part.